src/components/EmptyInfo.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+    <div v-if="emptyDashboard || emptyBoard" class="flex flex-col items-center justify-center">
+        <p class="mb-[25px] text-lg text-medium-grey">
+            {{ emptyDashboard ? 'Your dashboard' : 'This board' }}
+            is empty. Create a new
+            {{ emptyDashboard ? 'board' : 'column' }}
```

Comment:
move to computed, use css to make first letter uppercased

Reply:
**Done**